src/support/err.c

```diff
@@ -436,6 +436,31 @@ __wt_ext_err_printf(WT_EXTENSION_API *wt_api, WT_SESSION *wt_session, const char
     return (ret);
 }
 
+/*
+ * __wt_failpoint --
+ *     A generic failpoint function, it will return EBUSY if the failpoint triggers. Takes a double
+ *     representing the probability of the failpoint occurring. Supports percentages with two
+ *     decimal places.
+ */
+int
+__wt_failpoint(WT_SESSION_IMPL *session, uint64_t conn_flag, double probability)
```

Comment:
my preference is to have return value as bool, whether it is time to fail or no, let the caller decide about the error code so that we can use the same fail point to simulate various different error cases, not necessarily only EBUSY.

Reply:
Agreed, I'll go with that.